app/Libraries/Markdown/OsuMarkdown.php

```diff
@@ -287,7 +287,10 @@ private function getHtmlConverterAndExtension(): array
     private function getIndexableConverter(): MarkdownConverter
     {
         if ($this->indexableConverter === null) {
-            $environment = $this->createEnvironment();
+            $extraConfig = $this->osuExtensionConfig;
+            // remove attributes.
+            $extraConfig['attributes_allowed'] = [];
```

Comment:
shouldn't this only be when it's set? And does it even matter what's being allowed in this context? Removing this line doesn't seem to change test result (also the comment can be more descriptive on the why...)

Reply:
seems like I had some weird combination of settings when I did this part 🤔